src/igvfd/schemas/treatment.json

```diff
@@ -311,7 +312,17 @@
                 "selection",
                 "stimulation"
             ],
-            "description": "The intended purpose for treating the samples; Activation: treatment is known to activate a pathway in the biosample; Agonist: a substance which is known to initiate a physiological response when combined with a receptor; Antagonist: a substance that is known to interfere with or inhibits the physiological action of another; Control: treatment applied to a sample for control purposes; Differentiation: treatment that is applied to convert a less specialized cell to a more specialized cell; De-differentiation: treatment used to reprogram differentiated cells back to less determined cell states; Perturbation: treatment applied to the sample in order to study the effect of its application; Selection: treatment used to affect biosample in a way that can be used to distinguish cells and select for in the downstream steps; Stimulation: treatment applied to stimulate a cellular pathway.",
+            "enum_descriptions": {
+                "activation": "Treatment is known to activate a pathway in the biosample.",
```

Comment:
"...activate a pathway or modification in the biosample".

Reply:
fixed